lib/index.js

```diff
@@ -210,6 +211,11 @@ var validatorFromKey = {
         return null;
     },
 
+    'triton.network.public_ipv4': function validateTritonNetworkPublicIpv4(val) {
+        assert.string(val, 'val');
```

Comment:
Do we check the IPv4-address-validity here?  Or do we do it in the consumer(s)?

Reply:
That it validates in NAPI is good and answers my concern.

Re: your proposed optimization -> If it makes it down to NAPI anyway do we need to optimize?  There's a possible KISS violation if we do more here.

Given how many things depend on this, AND they access v1.4.0 (v1.4.1 is published on NPM, and its tag commit is disconnected from master/main/trunk ), I'm LOATHE to do more here until we correct other things here, including:

- Update this to v1.4.2
- See how consumers:
   - sdc-docker (of course)
   - triton-cmon
   - sdc-vmapi
   - triton-cns
   handle this change.
   
I mean, we can probably update the others to 1.4.1, but I worry about those other ones Doing Things Wrong afterwards if they need to go to v1.4.2.